<?xml version="1.0" ?>
<robot name="softleg_cart"  xmlns:xacro="http://ros.org/wiki/xacro">
    <ros2_control name="MoteusPi3Hat_Interface" type="system">
        <hardware>
            <plugin>pi3hat_hw_interface/MoteusPi3Hat_Interface</plugin>
            <param name="main_timeout">400000</param>
            <param name="can_timeout">20000</param>
            <param name="rcv_timeout">20000</param>
            <param name="attitude">0</param> 
        </hardware>
        <joint name="joint1"> <!-- hip -->
            <param name="id">1</param>
            <param name="bus">2</param>
            <param name="motor_transmission">9.0</param> <!-- trasmissione primo motore link-->
            <param name="sec_enc_transmission">2.96</param> <!-- trasmissione secondo motore2enc-->
        </joint>
        <joint name="joint2"> 
           <param name="id">2</param>
            <param name="bus">3</param>
            <param name="motor_transmission">9.0</param>
            <param name="sec_enc_transmission">2.87</param>
        </joint>
  
    </ros2_control>
</robot>
